### ISSUE TITLE
CSS and matplotlib issues

### ISSUE THREAD

**Issue by jarrodmillman** [scipy-lectures/scipy-lecture-notes#550]
Before
![screenshot](https://user-images.githubusercontent.com/123428/202281850-9830c487-0849-42fe-9002-2fde576e0966.png)

After
![screenshot](https://user-images.githubusercontent.com/123428/202282043-34178a9d-3200-4e4e-b16c-5c60e8e79f95.png)

I am not sure we need to completely match the previous look (especially since we may decide to move to different web infrastructure after the 2022.2 release. But some of these issues should probably be addressed:

- [x] fix bulleted list in prereq section
- [x] remove `:orphan:` (I think we added that in #549)
- [x] `Knowing your problem` in `Chapter contents` changed (maybe not a big deal if we don't fix it)
- [x] box border and colors for `Prerequistites` and `Chapter contents` (maybe not a big deal if we don't fix it)

**Comment by stefanv** [comment]
What is "before" and "after" here?

**Comment by jarrodmillman** [comment]
Before
![screenshot](https://user-images.githubusercontent.com/123428/202283856-fcbfa541-9e32-4dd2-bc7c-3cd437298d4c.png)

After
![screenshot](https://user-images.githubusercontent.com/123428/202284034-ddb8cc5d-4cdf-4830-8077-5fc989abdf85.png)

- [ ] plot of non-convex function is smaller than the convex function (they were the same size before).
- [ ] The heading `A non-convex function` isn't at the same height as `A convex function:` in the blue box below the figures.
- [ ] Not a new issue, but it looks like there is some sort of yellow background to the figures on my browser.

**Comment by jarrodmillman** [comment]
> What is "before" and "after" here?

Before is what it looked like before we started updating everything (sphinx, matplotlib, etc.). In particular, I am comparing the 2022.1 release:
- https://scipy-lectures.org/advanced/mathematical_optimization/index.html
with the version rendered for #549:
- https://output.circle-artifacts.com/output/job/44069bf2-b423-4452-a91f-d494eca01566/artifacts/0/build/html/advanced/mathematical_optimization/index.html

**Comment by jarrodmillman** [comment]
Before
![screenshot](https://user-images.githubusercontent.com/123428/202286159-36fa971c-6949-4be8-98f2-bed7a703b43e.png)

After
![screenshot](https://user-images.githubusercontent.com/123428/202286320-8c3e2498-8ef1-4afd-8322-9d07f7e3bb53.png)

- [ ] math not rendered on left-side of the figure in section 2.7.1.4
- [ ] math not rendered in text in section 2.7.2.1

**Comment by jarrodmillman** [comment]
Before (section 2.7.2.1)
![screenshot](https://user-images.githubusercontent.com/123428/202286785-578c1d8a-e4a7-4d1d-a1f2-5d744b979eb6.png)

After
![screenshot](https://user-images.githubusercontent.com/123428/202286950-99c5bf8c-9316-450d-833f-d31a5c102caa.png)

- [ ] Figures smushed

**Comment by jarrodmillman** [comment]
This is an example of something I think is improved, but noting it since it is different:

Before
![screenshot](https://user-images.githubusercontent.com/123428/202287580-83fb96df-a88e-4bde-9cf0-9959ee9dd67e.png)

After
![screenshot](https://user-images.githubusercontent.com/123428/202288439-d53f1a41-9b07-4f9d-8ec3-954633ad530d.png)

Compare bottom plots (i.e., `An ill-conditioned very non-quadratic function` plots).

**Comment by stefanv** [comment]
>     * [ ]  math not rendered on left-side of the figure in section 2.7.1.4
>     * [ ]  math not rendered in text in section 2.7.2.1

I don't know why this happens sometimes, but doing a clean build fixes it for me.

**Comment by stefanv** [comment]
>     * [ ]  Not a new issue, but it looks like there is some sort of yellow background to the figures on my browser.

I'm going to leave this one for now. A visual barrier between the images helps to locate the two columns by eye. A light yellow background isn't the best way to do that, but I don't have a good idea of what to do there right now.

**Comment by stefanv** [comment]
- [ ] Latex math rendering is too big

**Comment by stefanv** [comment]
> This is an example of something I think is improved, but noting it since it is different:
>
> ...
> 
> Compare bottom plots (i.e., `An ill-conditioned very non-quadratic function` plots).

Are you referring to the vertical centering of text? I agree that the new version looks better.
